Code/Framework/AzFramework/AzFramework/TargetManagement/TargetManagementComponent.cpp

```diff
@@ -20,13 +22,19 @@
 #include <AzCore/UserSettings/UserSettings.h>
 #include <AzCore/std/parallel/lock.h>
 #include <AzCore/Debug/Profiler.h>
-#include <GridMate/Session/LANSession.h>
-#include <GridMate/Serialize/Buffer.h>
-#include <GridMate/Replica/ReplicaFunctions.h>
+#include <AzNetworking/Framework/INetworkInterface.h>
+#include <AzNetworking/Framework/INetworking.h>
```

Comment:
You could add these interface headers to an AzNetworking.API target that is exposed in the AzNetworking CMakeLists.txt